setup.py

```diff
@@ -135,20 +135,23 @@ def get_data_files(name, version, fullname):  # pylint: disable=R0912
         set_systemd_files(data_files, dest=systemd_dir_path,
                           src=["init/mariner/waagent.service"])
     elif name == 'ubuntu':
-        set_bin_files(data_files, dest=agent_bin_path)
         set_conf_files(data_files, src=["config/ubuntu/waagent.conf"])
         set_logrotate_files(data_files)
         set_udev_files(data_files)
         if version.startswith("12") or version.startswith("14"):
             # Ubuntu12.04/14.04 - uses upstart
+            if version.startswith("12"):
```

Comment:
I'm thinking we should probably not change anything for ubuntu 14, just in case

The issue with Ubuntu 20 is that "python" is gone, but it should be there on 14

Reply:
In the DCR, packer using py3 template from ubuntu 14 and debian too.  So we decided to have setup handle this for all distros that need it and remove this from our DCR provisioners.

Reply:
Anyway we should have python until 20. So old agent file with python shebang still works. Are you asking only for 14 or until 20?

Reply:
I was asking for Ubuntu 14 only. So the DCR provisioner for ubuntu 14 was changing the shebang?

Reply:
Yes we are.. Same template from ubuntu 14

Reply:
k